plugins/package-managers/conan/src/funTest/assets/projects/synthetic/conan2-expected-output-py.yml

```diff
@@ -0,0 +1,337 @@
+---
```

Comment:
Commit message:
- "A new handler has been created" -> "Create a new handler to support..." (Stick to present tense and passive voice.)
- "Conan 2 returns more packages in the dependency tree ~as~ *than* Conan 1."
- "same time: only" -> "same time: Only" (Capitalize after colons.)
- "with add a parameter" -> "*will* add a parameter"